### PR TITLE
Update REAMDE pt-BR

### PR DESCRIPTION
**Pull request by JefersonLucas**

### CODE REVIEW

File: translations/pt-BR/README.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ vários formatos que poderiam ser usados; um exemplo é mostrado abaixo.
 ### Pré-processadores: considerações de formatação adicionais
 
 Diferentes pré-processadores de CSS possuem diferentes características,
-funcionalidades e sintaxe. Suas convenções devem ser extendidas para acomodar
+funcionalidades e sintaxe. Suas convenções devem ser estendidas para acomodar
```
Comment (hrag) [inline translations/pt-BR/README.md:257]: I don't speak Portuguese or anything, but I checked these words and both mean the same: "extended"

Is there some other difference between the two? Curious at this point.

Comment (jeremydouglass) [reply translations/pt-BR/README.md:257]: EXTENDER is common but incorrect. Three out of three online Portuguese language teachers agree -- use S (ESTENDER).

-  https://duvidas.dicio.com.br/estendido-ou-extendido/
-  https://portuguessemmisterio.com.br/2019/08/21/extender-ou-estender-extendido-ou-estendido-extensao-ou-estensao/
-  http://g1.globo.com/educacao/blog/dicas-de-portugues/post/extender-ou-estender.html

...however, always use X for extension / extensão.
`¯\_(ツ)_/¯`

Comment (JefersonLucas) [reply translations/pt-BR/README.md:257]: Portuguese is a very curious language, sometimes even we Brazilians don't understand it, but there is a word with a similar meaning as "extension" with 'x' that we use to plug in cables. I used the word extended for reasons of regionalization in Brazil. I hope I've helped!

Comment (hrag) [reply translations/pt-BR/README.md:257]: Ha! I completely understand - English is just as weird.
I'll take you guys' word for it!

Comment (JefersonLucas) [reply translations/pt-BR/README.md:257]: The Portuguese language is very complex, even more so each region of Brazil has different accents, but this is the correction.

**Pull request closed without merging**